js/utils/CommonUtils.js

```diff
@@ -186,6 +192,16 @@ define([
 	const f = (a, b) => [].concat(...a.map(d => b.map(e => [].concat(d, e))));
 	const cartesian = (a, b, ...c) => (b ? cartesian(f(a, b), ...c) : a);
 
+	const toggleCheckbox = function(hasPermissions, selectedConcepts, d, field, successFunction) {
```

Comment:
Just a question, maybe let's rename this to smth like toggleConceptSetCheckbox? If it's commonUtils so it's supposed that this functions would fit to everything.

Reply:
Thanks for that feedback @vlbe - I agree that making those functions more descriptive would be best since they reside in a common utility class. I've made that change in https://github.com/OHDSI/Atlas/pull/1852/commits/f0214db8b2e6e9d2d8d97d3b1f71b159afab65cd